symphony/content/content.blueprintsdatasources.php

```diff
@@ -168,7 +168,7 @@ public function __form()
 
                             // Handle pre Symphony 2.2.1 Static DS's
                         } else {
-                            $fields['static_xml'] = trim($existing->grab());
+                            $fields['static_xml'] = trim($existing->execute());
```

Comment:
This code block runs on DS's generated in Symphony 2.2.1, where `execute` was not deprecated. I don't think we should make this change.